### PR TITLE
[3.9] community/openjdk7: security upgrade to 7.221.2.6.18

### PR DESCRIPTION
**Pull request by J0WI**
https://github.com/alpinelinux/aports/pull/7928

### CODE REVIEW

File: community/openjdk7/APKBUILD [hunk 1]
```diff
@@ -88,6 +88,10 @@ source="https://icedtea.classpath.org/download/source/icedtea-$_icedteaversrc.ta
 
 # secfixes:
 #   7.211.2.6.17-r0:
```
Comment (bratkartoffel) [inline community/openjdk7/APKBUILD]: Wrong version?

Comment (J0WI) [reply community/openjdk7/APKBUILD]: Thanks, fixed

**Comment by J0WI** [comment]
:ping_pong: @fabled

**Comment by J0WI** [comment]
:ping_pong: @bratkartoffel

**Comment by fabled** [comment]
Committed in 996bdd5b5fed629c87f4d17cdf5cec5229eb7c72.

**Pull request closed without merging**